### PR TITLE
Ber or fol117 corr mk

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

File: Berlin/BerOrFol117.xml [hunk 1]
```diff
@@ -80,11 +85,262 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
...
+
+<msPart xml:id="p2">
+  <msIdentifier></msIdentifier>
+  <msContents>
+
+<msItem xml:id="p2.1">
+<locus from="1" to="26"/>
+<title xml:lang="en" type="complete" ref="LIT6187Dersan3Tahsas">Homily for the 3rd of Tāḫśāś on the birth of Mary, her presentation in the Temple,
+  marriage to Joseph and visit to Zacharias and Elizabeth</title>
+<note>Indication for reading: <q xml:lang="gez">አመ፫፡ ለተኅሣሥ፡</q></note>
```
Comment (thea-m) [inline Berlin/BerOrFol117.xml:125]: I think that here, `<foreign>` should be used, `<q> `is only for additions

File: Berlin/BerOrFol117.xml [hunk 2]
```diff
@@ -80,11 +85,262 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
...
+</msItem>
+
+  </msContents>
+
+<physDesc>
+<objectDesc form="Codex">
+<supportDesc>
+<support>
+<material key="parchment"/>
+</support>
```
Comment (thea-m) [inline Berlin/BerOrFol117.xml:289]: Could you give the number of leafs and their extent in the composite manuscript, and also information on foliation (if available)? In composite manuscripts, I find this always really helpful to get an impression of the structure.

File: Berlin/BerOrFol117.xml [hunk 3]
```diff
@@ -80,11 +85,262 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
...
+<additions>
+
+<list>
+
+<item xml:id="a1">
+<desc type="OwnershipNote"></desc>
+<q xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ። በአኰቴተ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ዛቲ፡ መጽሐፍ፡ ተፈነወት፡ እም፡ ሐበ፡ ማኅየዊ፡ እግዚእ፡ ፈላሲ፡
+ትብጻሕ፡ ኢየሩሳሌም፡ ለእመ፡ በጸሕኩ፡ ነፍስየ፡ እብእ፡ ወለእመ፡ ኢበጸሕኩ፡ አነ፡ እለ፡ ብጽሑ፡ ያብእዋ፡ ወእለ፡ ኮኑ፡ ቀሲሳን፡ ለዛቲ፡ መካን፡ ከመ፡ ይዘክሩኒ፡ እመኒ፡ ሞትኩ፡
+ወእመኒ፡ ሐየውኩ፡ ሰላም፡ ለክሙ፡ ወበጸሎትክሙአ፡ ኢትርስዕኒ።</q>
+<note>The note states that the manuscript was sent by a monk <persName ref="PRS13893Mahyawi">Maḥyawi ʾƎgziʾ</persName> to <placeName ref="LOC3957Jerusa">Jerusalem</placeName>.</note>
```
Comment (thea-m) [inline Berlin/BerOrFol117.xml:324]: It is better to move this information inside `<desc>`, then note can be deleted

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you, just some small things

**Comment by MarcinKrawczuk** [comment]
Thank you, I hope it's better now regarding the pagination, also I added
one more note and a bit of info on rubrication

wt., 25 paź 2022 o 13:10 thea-m ***@***.***> napisał(a):

**Review by thea-m — APPROVED** [review]
Thank you!

**Comment by eu-genia** [comment]
note that the merged file does not validate; it has an old file and master branch version as the basis and the git was not rebased before the push. work is needed here. i suggest that the push is reverted so that we do not have problematic files and then resubmitted.

all old branches created before the new validation has been put in place should be treated with care and preferably rebased before merging

**Comment by eu-genia** [comment]
PS there is of course no problem in working with Berlin manuscripts if these are needed for personal research / other projects, but they are not a Bm priority as the Qalamos project is supposed to be processing existing catalogues of mss in German libraries

**Comment by eu-genia** [comment]
as far as I can see the main problem is the xml:id of msItems (should be `p1_i1 `not `p1.i1` etc)
Thank you!

**Comment by MarcinKrawczuk** [comment]
So should I do a modification and correct  p1.i1 into  p1_i1 etc.?

śr., 26 paź 2022 o 22:52 Eugenia ***@***.***> napisał(a):

**Comment by eu-genia** [comment]
whatever it takes to have the file valid, I did not check if there  were more validation errors. thank you

**Comment by eu-genia** [comment]
(please create a new branch)

**Comment by eu-genia** [comment]
fixed now https://github.com/BetaMasaheft/Manuscripts/pull/1916

**Pull request merged**